src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java

```diff
@@ -439,6 +465,19 @@ private void updateCredentialsFile(String profileName, String awsAccessKey, Stri
             }
         }
     }
+    private void addOrUpdateProfile(String profileName,String oktaSession ,Instant start) throws IOException {
+
+        String profileStore = getMultipleProfilesIniPath().toString();
+
+        try (final Reader reader = profileStore.isEmpty() ?
```

Comment:
profileStore is just the path to the file, looks like this will never be empty based on getMultipleProfilesIniPath